lib/cocina_display/concerns/url_helpers.rb

```diff
@@ -44,6 +44,17 @@ def iiif_manifest_url(version: 3)
         "#{purl_url}/#{iiif_path}/manifest" if purl_url.present?
       end
 
+      # The Searchworks URL for this object.
+      # Uses the catkey (FOLIO HRID) if present, otherwise uses the druid.
+      # @note This does not guarantee that the object is actually in Searchworks.
+      # @return [String]
+      # @example
+      #  record.searchworks_url #=> "https://searchworks.stanford.edu/view/bx658jh7339"
+      def searchworks_url
```

Comment:
Is this something we're going to be indexing somewhere?

Reply:
Yeah, Earthworks uses it to point to the Searchworks version of a record.